backend/public-graph/graph/resolver.go

```diff
@@ -834,6 +835,25 @@ func (r *Resolver) HandleErrorAndGroup(ctx context.Context, errorObj *model.Erro
 		}
 	}
 
+	withinBillingQuota, quotaPercent := r.IsWithinQuota(ctx, pricing.ProductTypeErrors, workspace, time.Now())
+	go func() {
+		defer util.Recover()
+		if workspace.PlanTier != privateModel.PlanTypeFree.String() && workspace.AllowMeterOverage {
+			if quotaPercent >= 1 {
+				if err := model.SendBillingNotifications(ctx, r.DB, r.MailClient, email.BillingErrorsUsage100Percent, workspace); err != nil {
+					log.WithContext(ctx).Error(e.Wrap(err, "failed to send billing notifications"))
+				}
+			} else if quotaPercent >= .8 {
+				if err := model.SendBillingNotifications(ctx, r.DB, r.MailClient, email.BillingErrorsUsage80Percent, workspace); err != nil {
+					log.WithContext(ctx).Error(e.Wrap(err, "failed to send billing notifications"))
+				}
+			}
+		}
+	}()
```

Comment:
would be nice to share this logic with the logs overage notifications logic (especially things like the `0.8` and `1` thresholds)